VagrantProvisionVars.sh

```diff
@@ -21,4 +26,4 @@ export RVM_BASE_URL=https://github.com/rvm/rvm
 export RVM_BINARIES_URL="${RUBY_BASE_URL}/binaries"
 export RVM_HOME="${RVM_HOME:-${HOME}/.rvm}"
 export RVM_VERSION=1.29.3
-export WITH_RVM=yes
+export WITH_RVM="${WITH_RVM:-yes}"
```

Comment:
Does this line allow the RVM install to be controlled via an Environment variable?

Reply:
Exactly.  This is so that if we set the `WITH_RVM` (or `WITH_HADOOP`) in Jenkins it'll take precedence over the default value of `yes`.